repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/json/JsonSerializer.java

```diff
@@ -310,6 +332,10 @@ public void serializeData(ProjectInspector inspector) {
             case PATCHED:
                 outputHasBeenPatchedInspector(inspector);
                 break;
+                
+            case FIXERBUILD:
+            	outputFixerBuildInspector(inspector);
```

Comment:
I'll manage this after getting your approval about the other things that I've just committed ( 	**Changes on the pipeline creation process and execution** commit).